magenta-lib/src/main/scala/magenta/tasks/AWS.scala

```diff
@@ -234,6 +234,16 @@ object ASG {
   case class TagExists(key: String) extends TagRequirement
   case class TagAbsent(key: String) extends TagRequirement
 
+  def getGroupByName(name: String, client: AutoScalingClient, reporter: DeployReporter): AutoScalingGroup = {
```

Comment:
This has been reinstated from https://github.com/guardian/riff-raff/pull/650/files#diff-ea994807827af4586d5d53f01370f437dcf113cbc40094b45272a6131cbb1673L234-L242